virtcontainers/api.go

```diff
@@ -77,6 +77,17 @@ func createSandboxFromConfig(ctx context.Context, sandboxConfig SandboxConfig, f
 		return nil, err
 	}
 
+	// Move runtime to sandbox cgroup so all process are created there.
+	if err := s.retrieveCgroupPath(); err != nil {
+		return nil, err
+	}
+	// Join to sandbox cgroup before create any asset so all will be created there
```

Comment:
'Join the sandbox cgroup before creating any processes, so they are all created within that cgroup'

^ -- does that make more sense?